common/onepanel/base/vars.yaml

```diff
@@ -21,9 +21,26 @@ database:
     required: true
     default: postgres
 application:
-  webHost:
+  host:
     required: true
     default: localhost
-  webPort:
-    required: true
-    default: 80
\ No newline at end of file
+  local:
+    webHTTPPort:
```

Comment:
should we call this `ui` or `web`?

Reply:
Now that you mention it, I'm leaning toward ui. Seems more consistent with api.